notification/README.md

```diff
@@ -1,5 +1,5 @@
 # Notification Module
-![Docker Pulls](https://img.shields.io/docker/pulls/commercetools/commercetools-adyen-integration-notification)
+[![Docker Pulls](https://img.shields.io/docker/pulls/commercetools/commercetools-adyen-integration-notification)](https://hub.docker.com/r/commercetools/commercetools-adyen-integration-notification)
```

Comment:
It would be nice to see the travis build badge here as well.

Reply:
It's only 1 build, and you could find it in the main readme.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/329/commits/d2fcc7b18cdd4a16494fadf80b19d50e4493575d